test/unit/Sql/ExpressionTest.php

```diff
@@ -179,4 +179,21 @@ public function testNumberOfReplacemensConsidersWhenSameVariableIsUsedManyTimes(
         $expression = new Expression('uf.user_id = :user_id OR uf.friend_id = :user_id', ['user_id' => 1]);
         $expression->getExpressionData();
     }
+
+    public function testConstructorWithFalsyValidParameters()
+    {
+        $falsyParameters = ['0', false, 0, [], 0.0];
```

Comment:
I think it will be better to use data provider.
Also you've missed empty string here.